packages/@react-spectrum/color/stories/ColorArea.stories.tsx

```diff
@@ -120,3 +147,81 @@ XBlueYGreenSize3000.args = {...XBlueYGreen.args, size: 'size-3000'};
 export let XBlueYGreenSize600 = Template.bind({});
 XBlueYGreenSize600.storyName = 'RGB xChannel="blue", yChannel="green", size="size-600"';
 XBlueYGreenSize600.args = {...XBlueYGreen.args, size: 'size-600'};
+
+export let XSaturationYLightness = Template.bind({});
+XSaturationYLightness.storyName = 'HSL xChannel="saturation", yChannel="lightness"';
+XSaturationYLightness.args = {xChannel: 'saturation', yChannel: 'lightness', defaultValue: 'hsl(0, 100%, 50%)', onChange: action('onChange'), onChangeEnd: action('onChangeEnd')};
+
+export let XLightnessYSaturation = Template.bind({});
+XLightnessYSaturation.storyName = 'HSL xChannel="lightness", yChannel="saturation"';
+XLightnessYSaturation.args = {xChannel: 'lightness', yChannel: 'saturation', defaultValue: 'hsl(0, 100%, 50%)', onChange: action('onChange'), onChangeEnd: action('onChangeEnd')};
+
+/* TODO: what does a disabled color area look like? */
+export let XSaturationYLightnessisDisabled = Template.bind({});
+XSaturationYLightnessisDisabled.storyName = 'HSL xChannel="saturation", yChannel="lightness", isDisabled';
+XSaturationYLightnessisDisabled.args = {...XSaturationYLightness.args, isDisabled: true};
+
+export let XHueYSaturationHSL = Template.bind({});
+XHueYSaturationHSL.storyName = 'HSL xChannel="hue", yChannel="saturation"';
+XHueYSaturationHSL.args = {xChannel: 'hue', yChannel: 'saturation', defaultValue: 'hsl(0, 100%, 50%)', onChange: action('onChange'), onChangeEnd: action('onChangeEnd')};
+
+export let XSaturationYHueHSL = Template.bind({});
+XSaturationYHueHSL.storyName = 'HSL xChannel="saturation", yChannel="hue"';
+XSaturationYHueHSL.args = {xChannel: 'saturation', yChannel: 'hue', defaultValue: 'hsl(0, 100%, 50%)', onChange: action('onChange'), onChangeEnd: action('onChangeEnd')};
+
+/* TODO: what does a disabled color area look like? */
+export let XHueYSaturationHSLisDisabled = Template.bind({});
+XHueYSaturationHSLisDisabled.storyName = 'HSL xChannel="hue", yChannel="saturation", isDisabled';
+XHueYSaturationHSLisDisabled.args = {...XHueYSaturationHSL.args, isDisabled: true};
```

Comment:
Good question, maybe something to bring up to spectrum? I think making everything gray (like what you have) is probably easiest since we won't have to worry about if the disabled handle is too hard too see and it maybe confusing if we only change the handle style but leave the color area filled in